js/main.js

```diff
@@ -22,6 +22,7 @@ function formSubmit(event) {
   elImgURL.setAttribute('src', '/images/placeholder-image-square.jpg');
   switchView('entries');
   elEntryForm.reset();
+  location.reload();
```

Comment:
`location.reload()` is not allowed to be used, it refreshes the page entirely, which is extremely resource-intensive. You want to make a DOM tree for the new entry and add it to the beginning of the `ul` using JavaScript, not by reloading the page.